lib/importer/index.js

```diff
@@ -67,6 +68,11 @@ exports.Initialise = (config, router, prototypeKit) => {
     return false;
   }, {})
 
+  prototypeKit.views.addFunction("importerSuggestHeaders", (data, start, count) => {
+    const session = data[IMPORTER_SESSION_KEY];
+    return sheets_lib.GetRows(session, start, count)
+  }, {});
```

Comment:
I'm not quite sure of the meaning of "suggest" here... are we envisioning some more advanced logic to auto detect headers? If so, can we add a comment that suggests that?

If so, why does it take a `count`... shoudn't it just take the data as input and suggest headers based on that?

Reply:
The data passed in here is the request session data in the template, not the row data.  Trying to avoid having the templates juggling data and calling functions because non of it persists past the template lifetime.  

importerSuggestHeaders just a bad choice of name, we're asking for the first 10 rows (when looking for the header) so I must have been thinking it's suggesting the header is in these rows.  Perhaps importerGetRows?